build.sbt

```diff
@@ -327,7 +328,8 @@ lazy val usesMacros = Seq(
 
 lazy val libManagedSettings = Seq(
   genManaged := {
-    (Compile / genProps).value ++ (Compile / genSchemas).value
+    (Compile / managedSources).value
+    (Compile / managedResources).value
```

Comment:
I was confused by not seeing any definition of `managedSources managedResources`, so I read the sbt documentation.  I think here's what's going on.  Line 22 above declares `genManaged` is a sbt task which we can type at the sbt prompt.  I don't know the exact semantics of the `genManaged =` on line 22 and `genManaged :=` on line 330, but line 330 tells sbt that in order to evaluate `genManaged` (which has no value, but still needs to be evaluated), sbt must also get (and subsequently discard) the values of `managedSources managedResources` which are implicitly generated on the fly by calling the `sourceGenerators resourceGenerators` which we do define on lines 388 and 391.